src/main.rs

```diff
@@ -71,10 +74,14 @@ fn main() -> surf::Result<()> {
         } else {
             NetID::Mainnet
         };
+
+        println!("MelMinter v{} / melwalletd at {}", VERSION.unwrap_or("(N/A)"), daemon_addr);
```

Comment:
Melminter is spelled "Melminter" or "melminter", not "MelMinter"

say "internal melwalletd" instead, because it's not intended to refer to the external, main melwalletd